ci/tests.py

```diff
@@ -237,6 +237,155 @@ def test_generate_report(self):
         (file_name, data, mimetype) = report_email.attachments[0]
         self.assertEqual('report-2016-01-01-to-2016-02-01.xlsx', file_name)
 
+    @responses.activate
+    def test_generate_report_health_worker_registrations(self):
+        """
+        When generating a report, the second tab should be registrations per
+        health worker, and it should have the correct information.
+        """
+
+        # Registrations, first page, just returns empty results to make sure
+        # we're actually paging through the results sets using the `next`
+        # parameter
+        responses.add(
```

Comment:
We should maybe move adding the responses out into reusable functions if we are going to create a test function for each report tab